_templates/_page_openshift.html.erb

```diff
@@ -67,6 +67,14 @@
       </span>
       <% end %>
 
+      <% if (distro_key == "openshift-lightspeed) %>
```

Comment:
```suggestion
      <% if (distro_key == "openshift-lightspeed") %>
```

Reply:
thank you @kalexand-rh. I didn't see this file indicated in the logs.  I appreciate your help!